docs/walkthrough/index.md

```diff
@@ -0,0 +1,272 @@
+---
+layout: default
+title: Deploying Promitor, Prometheus, and Grafana on an AKS Cluster
+---
+
```

Comment:
Thinking out loud - Every section is now an h2, would it be easier to follow along if we would use different layers of headers?

For example:
1. Setting up the Azure infrastructure
	...
2. Deploying Promitor & Prometheus
	2.1 Creating a metrics declaration for Promitor
	...
3. Visualizing Metric
	...
4. Alerting
	...

Reply:
Yes, this is a good idea. I'll reorg when I have a minute 👍

Reply:
Thanks man! Don't worry, take your time!